hexstody-btc/src/worker.rs

```diff
@@ -45,6 +47,26 @@ pub async fn node_worker(
     }
 }
 
+pub async fn cold_wallet_worker(
+    client: &Client,
+    tx_notify: Arc<Notify>,
+    cold_amount: Amount,
+    cold_address: Address
+) -> (){
+    loop {
+        tx_notify.notified().await;
+        if let Ok(bal) = client.get_balance(Some(3), None){
+            if bal > cold_amount {
+                let amount = if bal > (cold_amount + cold_amount) { bal - cold_amount } else {cold_amount};
```

Comment:
Forbidden knowledge of multiplication:
`bal > 2*cold_amount`

Reply:
Very funny. You see, 2 is not an Amount. And you can't multiply integer with Amount. So I thought it's easier to just make it cold_amount + cold_amount, rather than allocating a new Amount